src/PerimeterxCookie.php

```diff
@@ -2,191 +2,6 @@
 
 namespace Perimeterx;
 
-abstract class PerimeterxCookie {
+abstract class PerimeterxCookie extends PerimeterxPayload {
```

Comment:
As we discussed, they dont have distinct functionality now but that is likely to change in the future.